src/map/mob_modifier.h

```diff
@@ -27,77 +27,78 @@ This is a list of mob specific modifiers. They can be added to pools / families
+    MOBMOD_NO_DROPS            = 38, // If set monster cannot drop any items, not even seals.
+    MOBMOD_SHARE_POS           = 39, // share a pos with another mob (eald'narche exoplates)
+    MOBMOD_TELEPORT_CD         = 40, // cooldown for teleport abilities (tarutaru AA, angra mainyu, eald'narche)
+    MOBMOD_TELEPORT_START      = 41, // mobskill ID to begin teleport
+    MOBMOD_TELEPORT_END        = 42, // mobskill ID to end teleport
+    MOBMOD_TELEPORT_TYPE       = 43, // teleport type - 1: on cooldown, 2 - to close distance
+    MOBMOD_DUAL_WIELD          = 44, // enables a mob to use their offhand in attacks
+    MOBMOD_ADD_EFFECT          = 45, // enables additional effect script to process on mobs attacks
+    MOBMOD_AUTO_SPIKES         = 46, // enables additional effect script to process when mob is attacked
+    MOBMOD_SPAWN_LEASH         = 47, // forces a mob to not move farther from its spawn than its leash distance
+    MOBMOD_SHARE_TARGET        = 48, // mob always targets same target as ID in this var
+    MOBMOD_CHECK_AS_NM         = 49, // If set , mob will check as a NM
+    MOBMOD_PROC_2HOUR          = 50, // chance of mob's 2 hour activating 0-100% (OBSOLETE - use job_special mixin)
+    MOBMOD_ROAM_TURNS          = 51, // Maximum amount of turns during a roam
+    MOBMOD_ROAM_RATE           = 52, // Roaming frequency. roam_cool - rand(roam_cool / (roam_rate / 10))
+    MOBMOD_BEHAVIOR            = 53, // Add behaviors to mob
+    MOBMOD_GIL_BONUS           = 54, // Allow mob to drop gil. Multiplier to gil dropped by mob (bonus / 100) * total
+    MOBMOD_IDLE_DESPAWN        = 55, // Time (in seconds) to despawn after being idle
+    MOBMOD_HP_STANDBACK        = 56, // mob will always standback with hp % higher to value
+    MOBMOD_MAGIC_DELAY         = 57, // Amount of seconds mob waits before casting first spell
+    MOBMOD_SPECIAL_DELAY       = 58, // Amount of seconds mob waits before using first special
+    MOBMOD_WEAPON_BONUS        = 59, // Add a bonus percentage to mob weapon damage ( bonus / 100 )
+    MOBMOD_SPAWN_ANIMATIONSUB  = 60, // reset animationsub to this on spawn
+    MOBMOD_HP_SCALE            = 61, // Scale the mobs max HP. ( hp_scale / 100 ) * maxhp
+    MOBMOD_NO_STANDBACK        = 62, // Mob will never standback
+    MOBMOD_ATTACK_SKILL_LIST   = 63, // skill list to use in place of regular attacks
+    MOBMOD_CHARMABLE           = 64, // mob is charmable
+    MOBMOD_NO_MOVE             = 65, // Mob will not be able to move
+    MOBMOD_MULTI_HIT           = 66, // Mob will have as many swings as defined.
+    MOBMOD_NO_AGGRO            = 67, // If set, mob cannot aggro until unset.
+    MOBMOD_ALLI_HATE           = 68, // Range around target to add alliance member to enmity list.
+    MOBMOD_NO_LINK             = 69, // If set, mob cannot link until unset.
+    MOBMOD_NO_REST             = 70, // Mob cannot regain hp (e.g. re-burrowing antlions during ENM).
```

Comment:
Just for clarification all these simply got their padding increased by one column to improve verbosity of the new modifier at position 71.

Reply:
if you could infill the "missing" ones into status.lua so we don't mistake them for having being free like the comment incorrectly says, I would appreciate it.

Reply:
See c3aaaa415a3c6a82d71503f95e22af45aefc591e

Reply:
After conversation the new mod now occupies again slot 13, obsolete mods have been removed and two TODO mods marked as such. A comment has been added to the source with the mirroring table and a comment about always editing both. See 611a174349f77bbd594a77881d7e528ed8da0b64

Reply:
After dialogue: fixed only for the affected lines by this PR for cleanliness of the PR itself. See ff24165b3b3b2dcc6f2b07f85ffad18954bbdbdb